### DISCUSSION TITLE
notifications and translations

### DISCUSSION THREAD

**Discussion by nassirGH** [filamentphp/filament#9765] [General]
I am currently working on customizing notifications and translations. As I navigate through this process, I find myself in need of further clarification on certain aspects.

I would greatly appreciate any guidance or insights the community can provide on customizing notifications and translations effectively. If possible, I would like to initiate a discussion or receive feedback to ensure that our customization aligns with best practices and community recommendations. @danharrin

**Comment by danharrin** [comment]
In what way are you trying to customize them

**Comment by haythembenkhlifa** [comment]
I also didn't figure out how to add translation to a notification.
I have a notification called OrderCanceled and inside it, I send a database notification as follows :

```
 public function toDatabase(User $notifiable): array
 {
        return NotificationsNotification::make()
            ->title('order_canceled')
            ->actions([
                Action::make('view_order')
                    ->button()
                    ->url(OrderResource::getUrl('edit', ['record' => $this->order]))
            ])
          ->getDatabaseMessage();
 }
```

I published the filament-notifications-translations but didn't figure out how to add a translation to my notification. 
@danharrin can you please assist?

**Comment by e2d2-dev** [comment]
use the __() or trans() function

[https://laravel.com/docs/11.x/localization](url)
